c++/src/capnp/rpc.c++

```diff
@@ -1748,6 +1757,13 @@ private:
     }
 
     RemotePromise<AnyPointer> send() override {
+      //if (callBuilder.getIsRealtime()) {
+      //  // In a situation involving a call being proxied over another connection, it could
+      //  // happen that send() is called for a realtime stream. In that case, redirect to
+      //  // sendStreaming() directly.
+      //  return RemotePromise<AnyPointer>(sendStreaming(), getDisabledPipeline());
+      //}
```

Comment:
@kentonv: This does not compile, and I am a bit stuck. It is related to [your comment here](https://github.com/capnproto/capnproto/pull/1460/files#r1233122531).

It seemed fine to just delegate to `sendStreaming()` here, but I am not sure how to convert the `Promise<void>` of `sendStreaming()` into a `RemotePromise<AnyPointer>` (because it requires a `Response`).

Any insight here? :pray: